<?xml version="1.0" encoding="utf-8"?>
<launch>
	<arg name="camera_input_image_raw_topic_arg" default="/rgb/image_color" />
	<arg name="tagged_image_output_topic_arg" default="/rgb/image_color_tagged" />
	<arg name="depth_points_topic_arg" default="/depth/points" />
	<arg name="canny_threshold_min_arg" default="200" />
	<arg name="canny_threshold_max_arg" default="500" />
	<!--  -->
	<node pkg="object_recognition_node" type="object_recognition_node.py" name="object_recognition_node" output="screen" launch-prefix="">
		<param name="camera_input_image_raw_topic" value="$(arg camera_input_image_raw_topic_arg)" />
		<param name="tagged_image_output_topic" value="$(arg tagged_image_output_topic_arg)" />
		<param name="depth_points_topic_topic" value="$(arg depth_points_topic_arg)" />
		<param name="canny_threshold_min" value="$(arg canny_threshold_min_arg)" />
		<param name="canny_threshold_max" value="$(arg canny_threshold_max_arg)" />
	</node>
</launch>
